test/functional/api/window_spec.lua

```diff
@@ -220,8 +220,22 @@ describe('API/win', function()
       -- curwin didn't change back
       neq(win, curwin())
 
-      -- shows updated position after getchar() #20793
+      -- does not show updated position after getchar() in non-current window
```

Comment:
What does "getchar() in non-current window" mean?

Reply:
Wrong word ordering. It's the cursor that's set in a non-current window(after a getchar()).